pkg/awsutils/awsutils.go

```diff
@@ -625,37 +621,40 @@ func (cache *EC2InstanceMetadataCache) createENI() (string, error) {
 }
 
 func (cache *EC2InstanceMetadataCache) tagENI(eniID string) {
-	tagSvc := cache.tagSVC
-	arns := make([]*string, 0)
-
-	eniARN := &arn.ARN{
-		Partition: "aws",
-		Service:   "ec2",
-		Region:    cache.region,
-		AccountID: cache.accountID,
-		Resource:  "network-interface/" + eniID}
-	arnString := eniARN.String()
-	arns = append(arns, aws.String(arnString))
+	tags := []*ec2.Tag{
```

Comment:
Can you add a comment with the sample tag keys and values for an ENI??

Reply:
Yeah.